### ISSUE TITLE
Pointer events get passed to previous (invisible) month 

### ISSUE THREAD

**Issue by Gnito** [airbnb/react-dates#224]
Pointer events should be disabled when using enableOutsideDays (tested with single month DayPicker).
This is a problem if previous month has more week rows than current month. 

This could be fixed with simply adding ```pointer-events: none;```

CalendarMonth.scss:
```css
.CalendarMonth {
  text-align: center;
  padding: 0 13px;
  vertical-align: top;

  &:first-of-type {
    position: absolute;
    z-index: -1;
    opacity: 0;
    pointer-events: none;
  }
```

**Comment by majapw** [comment]
This seems reasonable to me. Also I just reproed and agree, it's weird and we should probably fix that. 

One thing is that maybe we should set `pointer-events` back to `auto` for `.CalendarMonth--animating` so that one can still interact with the calendar during the transition. Other than that I support this change.

**Comment by majapw** [comment]
Should be fixed in v4.3.0! Thanks!
